### ISSUE TITLE
Checks id field type(int) not enough

### ISSUE THREAD

**Issue by hugoviktor** [spotify/github-java-client#103]
I'm trying to use the checksApi, but when i try to create a checkRun I get the following error: 

com.fasterxml.jackson.databind.JsonMappingException: Numeric value (6971753714) out of range of int (-2147483648 - 2147483647)

this is because the CheckRunResponse class has the id field of int type, and the response id is greater than an int. i think I should be a String or Long.

**Comment by dziemba** [comment]
Yes this indeed seems like a bug in the library. Would you be willing to create a PR with a fix + test case?
